components/prism-bash.js

```diff
@@ -70,6 +70,8 @@
 		'operator': /&&?|\|\|?|==?|!=?|<<<?|>>|<=?|>=?|=~/,
 		'punctuation': /\$?\(\(?|\)\)?|\.\.|[{}[\];]/
 	};
+	
+	Prism.languages.shell = Prism.languages.bash;
```

Comment:
Could you move this down at the very bottom, just before `})(Prism);` please?

Reply:
Yep, thanks, done.